client/browser/src/shared/code-hosts/github/codeHost.module.scss

```
@@ -0,0 +1,110 @@
+@import 'wildcard/src/global-styles/breakpoints';
+
+@mixin github-dark-theme {
+    [data-color-mode='dark'] {
+        @content;
+    }
+
+    [data-color-mode='auto'] {
+        @media (prefers-color-scheme: dark) {
+            @content;
+        }
+    }
+}
+
+:root {
+    /* Map out CSS variables to the theme-aware GitHub ones */
+    --body-bg: var(--color-bg-canvas);
+    --border-color: var(--color-border-primary);
+    --secondary: var(--color-auto-gray-2);
+}
+
+@include github-dark-theme {
+    --mark-bg: var(--mark-bg-dark);
+}
+
+.command-palette-action-item--github {
```

Since these classes are in the CSS module, we don't need suffixes that ensure uniqueness.
Let's shorten them by removing repetitive parts. Applies to other scoped classes in this file.
```suggestion
.command-palette-action-item {
```